backend/api/src/test/kotlin/software/shonk/application/service/ShorkServiceTest.kt

```diff
@@ -290,4 +320,30 @@ class ShorkServiceTest {
             )
         )
     }
+
+    @Test
+    fun `game visualization data works`() {
```

Comment:
This doesn't really test if it "works" just that there is data. The Test name should reflect that imo